modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/samples/wsse/policy/jaspi/UsernamePasswordCallback.java

```diff
@@ -35,7 +35,7 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
    {
       WSPasswordCallback pc = (WSPasswordCallback)callbacks[0];
       if ("kermit".equals(pc.getIdentifier()))
-         pc.setPassword("thefrog");
+         pc.setPassword("therealfrog");
```

Comment:
Did all things https://github.com/jbossws/jbossws-cxf/commit/f275c43931b9113d393a2c02e82ace27e1d9029c back ported ?

Reply:
the value is in the property file was changed to "therealfrog" sometime in the past.  This is the correct value.

Reply:
This is the part of change in  https://github.com/jbossws/jbossws-cxf/commit/f275c43931b9113d393a2c02e82ace27e1d9029c.  Did the change in `SubjectCreator.java` backport to this PR ?

Reply:
Yes the code change from f275c43 is present in SubjectCreator.